packages/meteor-gazelle-createdby/lib/model.js

```diff
@@ -0,0 +1,32 @@
+Astro.createBehavior({
+  name: 'createdBy',
+  events: {
+    addbehavior: function (behaviorData) {
+      var self = this;
+
+      self.addFields({
+        createdBy: {
+          type: 'string',
+          default: null
+        },
+        updatedBy: {
+          type: 'string',
+          default: []
+        }
+      });
+
+      self.addEvents({
+        beforeInsert: function () {
+          self.createdBy = Meteor.userId();
```

Comment:
I'll change it to this since we're not referencing it again under different scope.